runtime/vm/oa64/unsafeHelper.s

```diff
@@ -0,0 +1,149 @@
+# Copyright (c) 2017, 2017 IBM Corp. and others
```

Comment:
The first date should be 1991 given it is a copy of runtime/vm/xa64/unsafeHelper.s which has that start date.

Reply:
I'm not sure what the correct date is, but it certainly isn't 1991.  This is recent addition, in the last couple of years.